_spinnaker_install_admin_guides/install-on-k8s.md

```diff
@@ -0,0 +1,710 @@
+ <li>Click on <b>Add inline policy</b>.</li>
+ <li>On the <b>JSON</b> tab, add the following snippet:</li>
+
+<pre class="highlight"><code>{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "s3:*",
+      "Resource": [
+        "arn:aws:s3:::spinnaker-abcxyz",
+        "arn:aws:s3:::spinnaker-abcxyz/*"
+      ]
+    }
+  ]
+}
+</code></pre>
+
+ <li>Click on <b>Review Policy</b>.</li>
+ <li>Give your inline policy a name, such as <code>s3-spinnaker-abcxyz</code>.</li>
+ <li>Click <b>Create Policy</b></li>
+</ol>
+</details>
+
+<details><summary>(Option 2) S3 using an IAM User</summary>
+
+<p>First, create the IAM user and grant it permissions on your bucket:</p>
+
+<ol>
+  <li>Log into the AWS Console (Web UI).</li>
+  <li>Navigate to the IAM Console. Click on <b>Services</b> > <b>Security, Identity, & Compliance</b> > <b>IAM</b>.</li>
+  <li>Click on <b>Users</b> on the left.</li>
+  <li>Click on <b>Add user</b>.</li>
+  <li>Give your user a distinct name based on your organization's naming conventions. This guide uses <code>spinnaker-abcxyz</code>.</li>
+  <li>Click on <b>Programmatic access</b>.</li>
+  <li>For this guide, do not add a distinct policy to this user. Click on <b>Next: Tags</b>. <i>You may receive a warning about how there are no policies attached to this user. You can ignore this warning.</i></li>
+  <li>Optionally, add tags, then click on <b>Next: Review</b>.</li>
+  <li>Click <b>Create user</b>.</li>
+  <li>Save the Access Key ID and Secret Access Key. You need this information later during Halyard configuration.</li>
+  <li>Click <b>Close</b>.</li>
+</ol>
+
+<p>Then, add an inline policy to your IAM user:</p>
+
+<ol>
+  <li>Click on our newly-created IAM user.</li>
+  <li>Click on <b>Add inline policy</b> (on the right).</li>
+  <li>On the <b>JSON</b> tab, add the following snippet:</li>
+
+<pre class="highlight"><code>{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "s3:*",
+      "Resource": [
+        "arn:aws:s3:::spinnaker-abcxyz",
+        "arn:aws:s3:::spinnaker-abcxyz/*"
+      ]
+    }
+  ]
+}
+</code></pre>
+<p>Replace <code>s3-spinnaker-abcxyz</code> with the name of your bucket.</p>
+  <li>Click on <b>Review Policy</b></li>
+  <li>Give your inline policy a name, for example <code>s3-spinnaker-abcxyz</code>.</li>
+  <li>Click <b>Create Policy</b></li>
+</ol>
+
+</details>
+
+<details><summary>Configure Spinnaker to access S3 with the IAM Role or User</summary>
+
+<p>Spinnaker needs information about which bucket to access.  Additionally, if you are using an IAM User to access the the bucket, Spinnaker needs credentials for the IAM User.</p>
+
+<pre class="highlight"><code># Update these snippets with the information for your bucket
+export BUCKET_NAME=spinnaker-abcxyz
+export REGION=us-west-2
+
+# This will prompt for the secret key
+hal config storage s3 edit \
+  --bucket ${BUCKET_NAME} \
+  --region ${REGION}
```

Comment:
i had to add --no-validate for this hal command to take. e.g.: 
hal config storage s3 edit \ 
   --bucket ${BUCKET_NAME} \ 
   --region ${REGION} \ 
  --no-validate

Reply:
Oh good call. If using an IAM user, the first command will fail validation cause it doesn't have creds.